### DISCUSSION TITLE
I created an Automated Alpine installer with ZFS and ZFSBootMenu

### DISCUSSION THREAD

**Discussion by psy0rz** [zbm-dev/zfsbootmenu#595] [General]
Since installing Alpine linux with ZFS and ZFSBootMenu requires a lot of steps, i've created an installer + basic image that has it all. 

Find it here: https://github.com/psy0rz/alpinebox

If people like it, perhaps we can add a reference the the wiki Alpine wiki page?

Edwin

**Comment by zdykstra** [comment]
Very nice! Is this script / setup (Alpine on ZFS, with ZFSBootMenu) being used in production environments at datux.nl?

**Comment by psy0rz** [comment]
Currently all our boxes run ZFS + Alpine, but with grub. Installation was usually done via internal manually created Alpine images. Those were created with ansible scripts which basically ended up being 90% shell code.

So we decided to transform it into 100% shell code on github, and  add ZfsBootmenu and some improvements. New boxes will be installed this way.

Hopefully more people will it. :)

(I'm also the creator of zfs-autobackup, so i already loved the idea of ZfsBootMenu.)

**Comment by psy0rz** [comment]
Since nobody likes magic binaries (including me), i've added some github automation.

So the image is now entirely built by github actions instead of manually on my desktop.
